view/frontend/web/js/view/cart/cardknox-google-pay.js

```diff
@@ -153,6 +159,32 @@
         var base_grand_total = (totals ? totals : quote)['base_grand_total'];
         return parseFloat(base_grand_total).toFixed(2);
     }
+    function isExistLastNameShippingAddress(data) {
+        var address = data.paymentData.shippingAddress;
+        var addressNameArray = []; 
+            addressNameArray = address.name.replace("[","").replace("]","").split(' ');
+        if (addressNameArray.length == 1 ) {
+            jQuery(".gpay-error").html("<div>Please check the shipping address information. Lastname is required. Enter and try again.</div>").show();
+            setTimeout(function () { 
+                jQuery(".gpay-error").html("").hide();
+            }, 4000);
+            throw "Please check the shipping address information. Lastname is required. Enter and try again.";
```

Comment:
@mstein-ck, 
This error comes when order placed from the cart page for the guest user, So I have added validation before placing the order, it will check last name is exist or not.

Reply:
I don't understand,  why can't you throw an `Error` instead of a `string`?
```suggestion
            throw new Error("Please check the shipping address information. Lastname is required. Enter and try again.");
```